native/metrodroid/metrodroid.xcodeproj/project.xcworkspace/xcshareddata/swiftpm/Package.resolved

```diff
@@ -3,7 +3,7 @@
     "pins": [
       {
         "package": "ZIPFoundation",
-        "repositoryURL": "https://github.com/weichsel/ZIPFoundation/",
+        "repositoryURL": "git@github.com:weichsel/ZIPFoundation.git",
         "state": {
           "branch": null,
           "revision": "edbeaa39b426e54702194b0a601342322f01e400",
```

Comment:
This appears to reference the same commit as before, despite the `XCSwiftPackageProductDependency` section changing; any idea about this?

Reply:
Possibly Xcode changing the repo URL for some reason? Might be a personal preference I set at some point, or because Xcode is authenticated for git pulls to my Github... Can revert if you want.

Reply:
Ah OK, I understand now.

I would preference HTTPS URIs – AFAIK pulling over SSH requires a GitHub account.  I don't think any of us have write access to this repository anyway! :)

Reply:
When I tried to load the project with this (where I _haven't_ set up GitHub credentials inside of XCode), I get errors about rejected credentials (because my SSH private key is encrypted).

XCode then _deletes_ `native/metrodroid/metrodroid.xcodeproj/project.xcworkspace/xcshareddata/swiftpm/Package.resolved`.

When I restore the file, it gets deleted again.

Rolling back this file to `master` (with the HTTPS URI) will again change it to the `git` URI, until I update `native/metrodroid/metrodroid.xcodeproj/project.pbxproj`.

[From reading Swift's forums, it sounds like this file is important to note a specific version of a library](https://forums.swift.org/t/package-resolved-should-go-in-the-gitignore/14699/6), but the dependency is declared as `0.9.9 + update to next major version`.  This is somewhat incompatible with the idea of making sure our build is reproducible, but seems to be a side-effect of Swift's package manager.

I fixed this in 67068b4c850c3ff35ca705dccc0fbe0a440dada8.

Reply:
👍